### PR TITLE
Header 1

### PR DESCRIPTION
**Pull request by VioletaScvortova**
Hi! Below is a preview of my first web page KMarketer. I think I should work on it a bit more, but I`ve tried.

I`ll be glad to receive your feedback and comments

Lesson 8  Homework  Violeta Scvortova

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width , initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>KMarketer: We Take Care of the Future</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <form action="/action_page.php" method="get">
```
Comment (IvanCeban) [inline index.html:13]: Why the from wraps the whole block? 
You don't need the form here at all as you don't have any form elements here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        <div class="introduction"> 
+    <header class="header">
+        <div class="wrapper">
+            <div class="header_wrapper">
+                <div class="header_logo">
```
Comment (IvanCeban) [inline index.html:18]: it seems like you have a bunch of unclosed divs here

Comment (IvanCeban) [reply index.html:18]: Check all your tags for closing tags. Seems like you accidentally removed them

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                    <a href="/" class="header_logo-link">
+                        <img src="./img/svg/KMarketer_Logo.svg" alt="We Take Care of the Future"
+                            class="header_logo-pic">
+                    </a>
+                </div>
+
+                <div>
```
Comment (IvanCeban) [inline index.html:25]: redundant div wrapper.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+</h1>
+<h4>We know how large objects will act, but things on a <br> 
+    small scale just do not act that way.</h4>
+
+<div class="buttons">
+    <button class="button-get-quote">Get Quote Now</button>
+    <button class="button-learn-more">Learn More</button>
+</div>
+</div>
+</div>
```
Comment (IvanCeban) [inline index.html:68]: these closing tags are not on their places. 
Remember the opening and closing tags should be always pared: `<div> <h1></h1></div>`

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,678 @@
...
+    resize: vertical;
+}
+
+.chromeframe {
+    margin: 0.2em 0;
+    background: #ccc;
+    color: #000;
+    padding: 0.2em 0;
+}
+
```
Comment (IvanCeban) [inline css/style.css:375]: seems like you duplicated the reset file here. No need to do that as you already have these reset styles in the reset.css file

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,678 @@
...
+.button-learn-more {
+    display: flex;
+    padding: 10px 36px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 10px;
+    border-radius: 37px;
+    border: 1px solid var(--light-background-color, #FFF);;
```
Comment (IvanCeban) [inline css/style.css:669]: redundant `;` which might break the following CSS code

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Good job! Nice css structuring. 
Good for using reset file. 

Pay more attention on HTML layout and get rid of redundant tags that make no effect on your page

**Comment by VioletaScvortova** [comment]
During my homework, I used this source and watched 3 videos for beginners: how to write code for a website using Figma 
https://www.youtube.com/watch?v=OkNfBnq_c7c&ab_channel=SteptoWeb

**Comment by VioletaScvortova** [comment]
@IvanCeban thank you a lot for your comments! Added updates. 

Lesson 8_update  Homework  Violeta Scvortova

**Review by Andrian-88 — APPROVED** [review]

File: test.html [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+        </div>
+        </div>
+    </form>
+</body>
+
+</html>
+
+
+
+header__icons
```
Comment (IvanCeban) [inline test.html:77]: seems like you forgot to remove these styles.

**Review by IvanCeban — APPROVED** [review]

File: test.html (hunk 7 above)
Comment (VioletaScvortova) [reply test.html:77]: @IvanCeban this is just test-file. Work-file is index.html

**Comment by VioletaScvortova** [comment]
@IvanCeban hi!  Here is the screenshot with responsive design:
Lesson 9_Media Queries  Homework  Violeta Scvortova

**Pull request merged**